daml-lf/language/daml-lf.bzl

```diff
@@ -5,11 +5,11 @@
 # If we make a new LF release, we bump latest and once we make it the default
 # we bump stable.
 lf_stable_version = "1.8"
-lf_latest_version = "1.8"
+lf_latest_version = "1.11"
```

Comment:
have we already discussed one we flip the default in the compiler? one release after it’s stable like we used to or has that changed with the preview scheme?

Reply:
I am not quite sure. 
@sofiafaro-da what do you think ? 
Should we track that in an issue ?

Reply:
I think the preview scheme is that we have one stable release before LF 1.11 becomes the default, and after that it becomes the default?

Reply:
sgtm so it will be the default in 1.11

Reply:
That's a nice coincidence -- that LF 1.11 becomes the default starting with SDK 1.11. Is there any chance we'd be able to keep new LF versions in sync with the SDK versions in which they become the default (by skipping a few version numbers ahead each time with the LF veresion)? Or is that a long shot?